crates/rustc_codegen_spirv/src/codegen_cx/mod.rs

```diff
@@ -66,6 +66,10 @@ pub struct CodegenCx<'tcx> {
 
     /// Simple `panic!("...")` and builtin panics (from MIR `Assert`s) call `#[lang = "panic"]`.
     pub panic_fn_id: Cell<Option<Word>>,
+    /// Intrinsic for loading a <T> from a &[u32]
+    pub buffer_load_intrinsic_fn_id: RefCell<FxHashSet<Word>>,
+    /// Intrinsic for storing a <T> into a &[u32]
+    pub buffer_store_intrinsic_fn_id: RefCell<FxHashSet<Word>>,
```

Comment:
Note to self: this is a set (and not just a `Cell<Option<_>>`) because it's recording SPIR-V IDs, not `DefId`s, so all of the monomorphizations are different (even if they share one `DefId`).

Reply:
ah yeah, thought about that when porting it over (was confused myself), really should have added a comment about that instead of copying stuff without thinking, haha